src/dashboard/src/settings/common.py

```diff
@@ -47,10 +50,31 @@
     }
 }
 
+# Memcached is the most efficient cache backend natively supported by Django.
+# However, Memcached is not available yet in the Archivetica stack but we may
+# consider adding it soon. Using local-memory caching for now so we can start
+# writing some cache-aware code.
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+    }
+}
+
 # Language code for this installation. All choices can be found here:
 # http://www.i18nguy.com/unicode/language-identifiers.html
 LANGUAGE_CODE = 'en-us'
 
+# https://docs.djangoproject.com/en/dev/ref/settings/#languages
+LANGUAGES = [
+    ('fr', _('French')),
+    ('en', _('English')),
+    ('es', _('Spanish')),
+]
```

Comment:
Is this the final set of languages, or just the ones for testing? If final, why these three, and not others?